src/components/cart/priceTable.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react'
+import { useCart } from '../../context/cartContext/cartContext';
+
+
+
+function PriceTable() {
+  const [bill, setBill] = useState({ priceOfItems: '', deliveryCharge: '', totalPrice: '', count: '' })
+
+
+
+  const { state } = useCart();
+
+  useEffect(() => {
+    //array of price values of all items in cart 
+    const arrayOfPriceOfItemsInCart = state.map(item => {
+      return Number(item.price) * Number(item.count)
+    })
+    // sum of all items ina cart
+    const sumOfItemsInCart = arrayOfPriceOfItemsInCart.reduce((previousValue, currentValue) => previousValue + currentValue, 0)
+
+    //array of count of all items in cart 
+    const arrayOfCountOfItemsInCart = state.map(item => {
+      return Number(item.count)
+    })
+    //count of all items in cart
+    const countOfItems = arrayOfCountOfItemsInCart.reduce((previousValue, currentValue) => previousValue + currentValue, 0)
+
+    //update bill
+    setBill({ priceOfItems: sumOfItemsInCart, deliveryCharge: 50, totalPrice: 50 + sumOfItemsInCart, count: countOfItems })
+  }, [state]);
+  console.log(bill)
```

Comment:
changes: remove console log when you push code to github

Reply:
ok,i have removed it thanks for the pr review